mdn/developer-portal
# 1716: Allow content pages to be featured pages on Homepage and Topic

This changeset permits content pages to be featured pages on Homepage and Topic pages, so that links to other pages within the site don't have to be done as ExternalPages (which opens a new tab despite the domain remaining the same, which is an a11y pain for users)

(Related issue #1716)

## Key changes:

- Support a 3:2 card image for a ContentPage, ahead of it becoming 'featureable' on the Homepage or a Topic page

- Allow ContentPages to be specificed as a featured page for Homepage and for Topic
  * Page config updated, including no-database-schema-change migrations
  * Featured card selector upgraded to allow showing contentpage as a featured card




## How to test

- Code will be deployed to DEV, not staging
- Confirm you can add a/any content page as a featured page on Homepage (previewing can be fine, you don't need to publish the homepage)
- as above, but with a Topic page
